health/health.d/exporting.conf

```
@@ -11,7 +11,11 @@ component: Exporting engine
      warn: $this > (($status >= $WARNING)  ? ($update_every) : ( 5 * $update_every))
      crit: $this > (($status == $CRITICAL) ? ($update_every) : (60 * $update_every))
     delay: down 5m multiplier 1.5 max 1h
-     info: number of seconds since the last successful buffering of exporting data
+     info: Number of seconds since the last successful buffering of exporting data. \
+           Exporting engine failed to buffer metrics for a while. \
+           Some metrics are lost while exporting. \
```

This should be past tense.

```suggestion
           Some metrics were lost while exporting. \
```